### DISCUSSION TITLE
Helmet  Detection Model Query

### DISCUSSION THREAD

**Discussion by ShraddhaMohoite** [ultralytics/yolov5#10447] [Q&A]
I have run the code for helmet detection, first I tried it by interfacing with 1 camera, by giving the IP address of that camera. The  it run correctly. But, when I tried the same model with 5 camera by giving their IP Addresses, then it gives the output with multiple overlapping screens. Can you help me for displaying all 5 screens without any overlapping among them. Also I want to capture snapshot of the person who is not wearing the helmet & wants to log all this data with our server. Please assist me for these queries.

**Comment by alijafari79** [comment]
> I have run the code for helmet detection, first I tried it by interfacing with 1 camera, by giving the IP address of that camera. The it run correctly. But, when I tried the same model with 5 camera by giving their IP Addresses, then it gives the output with multiple overlapping screens. Can you help me for displaying all 5 screens without any overlapping among them. Also I want to capture snapshot of the person who is not wearing the helmet & wants to log all this data with our server. Please assist me for these queries.

Are you using Python to get the Camera frames ?
Like using opencv to get the ip camera frames ?

**Comment by alijafari79** [comment]
> I have run the code for helmet detection, first I tried it by interfacing with 1 camera, by giving the IP address of that camera. The it run correctly. But, when I tried the same model with 5 camera by giving their IP Addresses, then it gives the output with multiple overlapping screens. Can you help me for displaying all 5 screens without any overlapping among them. Also I want to capture snapshot of the person who is not wearing the helmet & wants to log all this data with our server. Please assist me for these queries.

Are you using Python to get the Camera frames ?
Like using opencv to get the ip camera frames ?

**Comment by alijafari79** [comment]
> I have run the code for helmet detection, first I tried it by interfacing with 1 camera, by giving the IP address of that camera. The it run correctly. But, when I tried the same model with 5 camera by giving their IP Addresses, then it gives the output with multiple overlapping screens. Can you help me for displaying all 5 screens without any overlapping among them. Also I want to capture snapshot of the person who is not wearing the helmet & wants to log all this data with our server. Please assist me for these queries.

Would yo please give more info on how you get the frames and how you process to show them, to be able to guide you solve the problem ...
